README.md

```diff
@@ -1,2 +1,20 @@
 # hw01
 Repository for homework 01
+# About me
+Hi! My name is Jenny Xiao. I'm an undergraduate exchange student from the University of Hong Kong. I'm currently in my third year studying political science, American studies, and European studies. 
+My LinkedIn page: (https://www.linkedin.com/in/jenny-xiao-883952150/)
```

Comment:
Create an in-line style link in markdown like this: `[My LinkedIn page](https://www.linkedin.com/in/jenny-xiao-883952150/)`